Sources/_FoundationCShims/include/_CStdlib.h

```diff
@@ -122,9 +122,15 @@
 #include <complex.h>
 #endif
 
+#if !defined(_WIN32)
```

Comment:
If we need to special case windows here, can we use `TARGET_OS_WINDOWS` instead for consistency with the other headers?